### PR TITLE
[SAP CV] Update with Language edits for tutorials

### PR DESCRIPTION
**Pull request by kannankumar**
Updated tutorials with Language Edits and suggestions from @gabrielle-aus and @LunaticMaestro

### CODE REVIEW

File: tutorials/cv-package-aicore-deploy-object-detection/cv-package-aicore-deploy-object-detection.md [hunk 1]
```diff
@@ -11,35 +11,36 @@ author_profile: https://github.com/kannankumar
...
-- How to generate and deploy online inferencing server using computer vision package
-- How to consume endpoints of online inferencing server
+- How to generate serving templates for a computer vision package
+- How to generate and deploy an online inferencing server using a computer vision package
+- How to consume endpoints for online inferencing
 
 ## Prerequisites
- - You had trained model for object detection mentioned in [this tutorial](cv-package-aicore-train-object-detection).
- - You are using the notebook from the above mentioned tutorial.
+ - You have trained an AI model for object detection using the [set up tutorial](cv-package-aicore-train-object-detection).
```
Comment (MichaelCzcz) [inline tutorials/cv-package-aicore-deploy-object-detection/cv-package-aicore-deploy-object-detection.md:19]: Please remove full stop.

File: tutorials/cv-package-aicore-deploy-object-detection/cv-package-aicore-deploy-object-detection.md [hunk 2]
```diff
@@ -11,35 +11,36 @@ author_profile: https://github.com/kannankumar
...
+ - You are using the Jupyter notebook from the [set up tutorial](cv-package-aicore-train-object-detection).
```
Comment (MichaelCzcz) [inline tutorials/cv-package-aicore-deploy-object-detection/cv-package-aicore-deploy-object-detection.md:20]: Please remove full stop.

File: tutorials/cv-package-aicore-setup/cv-package-aicore-setup.md [hunk 3]
```diff
@@ -11,38 +11,45 @@ author_profile: https://github.com/kannankumar
...
-  - How to execute python code in Jupyter
+  - How to install Python packages for computer vision AI pipelines
+  - How to execute Python code in Jupyter
   - How to connect to SAP AI Core using the SAP AI Core SDK
 
 ## Prerequisites
   - You have completed the [Create Your First Machine Learning Project using SAP AI Core tutorial series](group.ai-core-get-started-basics)
   - You have [provisioned SAP AI Core](https://help.sap.com/viewer/2d6c5984063c40a59eda62f4a9135bee/LATEST/en-US/38c4599432d74c1d94e70f7c955a717d.html)
-  - You have [set up your Git Repository with SAP AI Core](https://help.sap.com/viewer/808d9d442fb0484e9b818924feeb9add/LATEST/en-US/3269092e37d141a293f0dbd7eaafc829.html).
+  - You have [set up your Git repository with SAP AI Core](https://help.sap.com/viewer/808d9d442fb0484e9b818924feeb9add/LATEST/en-US/3269092e37d141a293f0dbd7eaafc829.html).
```
Comment (MichaelCzcz) [inline tutorials/cv-package-aicore-setup/cv-package-aicore-setup.md:21]: Please remove full stop.

File: tutorials/cv-package-aicore-setup/cv-package-aicore-setup.md [hunk 4]
```diff
@@ -11,38 +11,45 @@ author_profile: https://github.com/kannankumar
...
+  - You have [registered your object store secret in SAP AI Core](https://help.sap.com/docs/AI_CORE/2d6c5984063c40a59eda62f4a9135bee/b083d73f672c428faac3048b74733546.html)
 
 ### Pre-read
+The computer vision package provides a fast and easy way to train computer vision models and set up a serving infrastructure. The computer vision package works with the SAP AI Core SDK (ai-core-sdk) to provide a command line interface to create SAP AI Core templates for training and serving.
+The package bundles capabilities of detectron2 (library for object detection and image segmentation) with extensions such as image classification, feature extraction and model serving.
+
+In this tutorial, you first create a Python virtual environment and install Python packages in the Python virtual environment. You then link the virtual environment to a Jupyter notebook so that you can run computer vision pipelines on SAP AI Core from a Jupyter notebook.
+
+After set up, you can train and serve an AI model for object detection. The aim of the model is to recognise objects (numbers) on electricity meter readings. See the related computer vision tutorials for training and serving an object detection model.
```
Comment (MichaelCzcz) [inline tutorials/cv-package-aicore-setup/cv-package-aicore-setup.md:31]: setup (to set sth. up, the setup)

File: tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md [hunk 5]
```diff
@@ -117,102 +136,100 @@ workflow_config = {
...
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 5: ](Create AI Core Template)]
+[ACCORDION-BEGIN [Step 5: ](Generate training template)]
 
+The workflows have another method `create-template` to build a training pipeline with placeholders for datasets and hyper-parameters. This training pipeline internally references the Docker image you created previously. You are not required to modify/ update the AI code contained, but you can tweak hyper-parameters to achieve different model qualities, demonstrated later in the tutorial.
 
-The `sap-cv` content package fetches the details from the workflow configuration (details of workflow, scenario, docker repo secrets, object store secrets, etc) and automatically generates a template for the selected workflow. You are using the `object-detection` workflow in this tutorial, so you generate an workflow for training an object detection model.
+Pase and edit the following snippet. Replace `<YOUR_GIT_Repo_PATH>` with the absolute path to the directory that contains the repository you've on-boarded to SAP AI Core. This repository syncs using Git Ops.
```
Comment (MichaelCzcz) [inline tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md:187]: Fix typo

File: tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md [hunk 6]
```diff
@@ -117,102 +136,100 @@ workflow_config = {
...
+After the template is pushed to the Git repo, you'll need to wait a few minutes for the template in the repo to sync with SAP AI Core. On-boarded Git repositories are regularly synced with SAP AI Core (~3 mins).
 
 [DONE]
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 6: ](Download data from public datasource)]
+[ACCORDION-BEGIN [Step 6: ](Download training data)]
+
+You'll need to use a dataset which contains images of electricity meters for this tutorial. The model will attempt to read the meters using object detection. Find more information about the dataset on [Gas-Meter Reading Datasets, Applied Recognition Technology Laboratory](http://artelab.dista.uninsubria.it/downloads/datasets/automatic_meter_reading/gas_meter_reading/gas_meter_reading.html)
```
Comment (MichaelCzcz) [inline tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md:228]: Missing comma before "which"

File: tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md [hunk 7]
```diff
@@ -331,45 +347,43 @@ except IndexError:
...
-
-- `executable_id`: Name of the workflow mentioned in workflow configuration.
-
-- `input_artifact_binding` - Binds the dataset artifact you created in previous tutorial and binds it as an Input Artifact for this workflow. [TODO: Explain succinctly]
-
-- `parameter_bindings` - Parameters for our Machine learning training you set in previous step.
+| Parameter | Description |
+| --- | --- |
+| `name` | Name of configuration |
+| `scenario_id` | Used to connect the configuration to the scenario ID mentioned previously in our workflow configuration |
```
Comment (MichaelCzcz) [inline tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md:363]: Please don't say "we", "us", "our", etc. unless you are referring to SAP.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/cv-package-aicore-deploy-object-detection/cv-package-aicore-deploy-object-detection.md (hunk 1 above)
Comment (kannankumar) [reply tutorials/cv-package-aicore-deploy-object-detection/cv-package-aicore-deploy-object-detection.md]: done

File: tutorials/cv-package-aicore-deploy-object-detection/cv-package-aicore-deploy-object-detection.md (hunk 2 above)
Comment (kannankumar) [reply tutorials/cv-package-aicore-deploy-object-detection/cv-package-aicore-deploy-object-detection.md]: done

File: tutorials/cv-package-aicore-setup/cv-package-aicore-setup.md (hunk 3 above)
Comment (kannankumar) [reply tutorials/cv-package-aicore-setup/cv-package-aicore-setup.md]: done

File: tutorials/cv-package-aicore-setup/cv-package-aicore-setup.md (hunk 4 above)
Comment (kannankumar) [reply tutorials/cv-package-aicore-setup/cv-package-aicore-setup.md]: done

File: tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md (hunk 5 above)
Comment (kannankumar) [reply tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md]: done

File: tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md (hunk 6 above)
Comment (kannankumar) [reply tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md]: done

File: tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md (hunk 7 above)
Comment (kannankumar) [reply tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md]: done

**Comment by kannankumar** [comment]
Corrected all punctuation and other errors identified in the review.

**Comment by gabrielle-aus** [comment]
Thanks @kannankumar for following up these points. And thanks @MichaelCzcz for your review - much appreciated

**Pull request merged**